pkg/garden/ted/glob.hoon

```diff
@@ -25,4 +25,14 @@
     =-  ?>(=(hash (sham -)) -)
```

Comment:
We might want to remove this hash check assertion while we're at it, `/app/docket` already handles this, and much more gracefully.

Reply:
do we still need the `|.` there?

Reply:
Yes, the `;;` might still crash.